beam-postgres/Database/Beam/Postgres/PgSpecific.hs

```diff
@@ -960,6 +960,14 @@ QExpr a ?| QExpr b =
 QExpr a ?& QExpr b =
   QExpr (pgBinOp "?&" <$> a <*> b)
 
+-- | Postgres @||@ operator. Concatenates two jsonb values into a new jsonb value.
+(||.) ::
+  QGenExpr ctxt Pg.Postgres s (Pg.PgJSONB a) ->
```

Comment:
nit: this arrow style is not the norm in Beam.

Reply:
Uh, totally right - fixed that (plus removing these `Pg.` prefixes`).